src/test/java/com/epam/qalab/DeliverToTest.java

```diff
@@ -0,0 +1,111 @@
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://www.amazon.com/");
+    }
+
+    @Test
+    public void verifyZipCodeUpdate() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        WebElement oldLocation = driver.findElement(By.id("glow-ingress-line2"));
+
+        deliverToButtonClick();
+
+        WebElement zipCodeField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("GLUXZipUpdateInput")));
+        zipCodeField.sendKeys("37219");
+
+        WebElement applyButton = driver.findElement(By.xpath("//*[@data-action='GLUXPostalUpdateAction']"));
+        applyButton.click();
+
+        WebElement continueButton = wait
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//div[@class='a-popover-footer']//input[@id='GLUXConfirmClose']")));
+        continueButton.click();
+
+        wait.until(ExpectedConditions.stalenessOf(oldLocation));
+        String actualLocation = driver.findElement(By.id("glow-ingress-line2")).getText();
+
+        Assert.assertTrue(actualLocation.contains("Nashville 37219"), "The 'Deliver to' location isn't updated.");
+    }
+
+    @Test
+    public void verifyPolandIsPresent() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        deliverToButtonClick();
+
+        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//span[@class='a-dropdown-container']")));
+        Select select = new Select(driver.findElement(By.xpath("//select[@class='a-native-dropdown']")));
+        List<String> list = new ArrayList<>();
+        for (WebElement element: select.getOptions()) {
+            list.add(element.getText());
+        }
+
+        Assert.assertTrue(list.contains("Poland"), "The list of countries doesn't contain Poland.");
+    }
+
+    @Test
+    public void verifyShippingToCountry() {
+        String country = "Bulgaria";
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        WebElement oldLocation = driver.findElement(By.id("glow-ingress-line2"));
+
+        deliverToButtonClick();
+
+        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//span[@class='a-dropdown-container']")));
+        Select select = new Select(driver.findElement(By.xpath("//select[@class='a-native-dropdown']")));
+        select.selectByVisibleText(country);
+
+        WebElement done = driver.findElement(By.xpath("//button[@name='glowDoneButton']"));
+        done.click();
+
+        wait.until(ExpectedConditions.stalenessOf(oldLocation));
+        WebElement category = driver.findElement(By.xpath("//a[@aria-label='Keyboards']"));
+        category.click();
+
+        WebElement keyboardItem = wait
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[contains(@class,'s-main-slot ')]/div[@data-component-type='s-search-result'][1]//h2/a")));
```

Comment:
Not critical, but it is better not to too long line. I mean the size of the line should be 120, no more. So it is better to move some part to a new line